cockatrice/src/carddatabase.cpp

```diff
@@ -1,20 +1,25 @@
 #include "carddatabase.h"
 #include "pictureloader.h"
 #include "settingscache.h"
+#include "spoilerbackgroundupdater.h"
 
 #include <QCryptographicHash>
 #include <QDebug>
 #include <QDir>
-#include <QDirIterator>
 #include <QFile>
-#include <QTextStream>
 #include <QMessageBox>
 
 const int CardDatabase::versionNeeded = 3;
 const char* CardDatabase::TOKENS_SETNAME = "TK";
 
 static QXmlStreamWriter &operator<<(QXmlStreamWriter &xml, const CardSet *set)
 {
+    if (! set)
```

Comment:
Can you explicitly check `nullptr` here rather than the implicit style please? It helps with clarity and intent

Reply:
And further down in the file also

Reply:
Yeah I'll fix that up